src/app/public/testing/phonefield/phone-field-fixture.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldFixture
+} from '@skyux/lookup/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX phone field component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyPhoneFieldFixture {
+  private _debugEl: DebugElement;
+
+  // this property is lazy loaded and should be accessed via the private countryFixture property
+  private _countryFixture: SkyCountryFieldFixture;
+
+  /**
+   * The value of the input field for the phone field.
+   */
+  public get inputText(): string {
+    return this.phoneFieldInput.value;
+  }
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this._debugEl = SkyAppTestUtility
+      .getDebugElementByTestId(fixture, skyTestId, 'sky-phone-field');
+
+    // The country selector needs extra time to initialize.
+    // Consumers shouldn't need to work around this so we do an extra detect here
+    fixture.detectChanges();
+  }
+
+  /**
+   * Sets the value of the input field for the phone field.
+   */
+  public async setInputText(inputText: string): Promise<any> {
+    const inputEl = this.phoneFieldInput;
+    inputEl.value = inputText;
+
+    SkyAppTestUtility.fireDomEvent(inputEl, 'input');
+    this.fixture.detectChanges();
+
+    SkyAppTestUtility.fireDomEvent(inputEl, 'change');
+
+    this.fixture.detectChanges();
+    return this.fixture.whenStable();
+  }
+
+  /**
+   * Opens the country selector, peforms a search, but makes no selection.
+   * @param searchText The name of the country to select.
+   */
+  public async searchCountry(searchText: string): Promise<NodeListOf<HTMLElement>> {
+    await this.openCountrySelection();
+
+    const countryFixture = await this.getCountryFixture();
```

Comment:
This will re-instantiate the `SkyCountryFieldFixture` each time. If a consumer runs `searchCountry` multiple times in the same test, it will create multiple instances. Should we just move the `new SkyCountryFieldFixture()` logic to its own encapsulated function and just call it once in the constructor?

Reply:
I don't see how this creates a new instance each time it is accessed. It will be created the first time it's accessed and then be cached in the private variable thereafter. 

I am lazy loading the property to work around the fact that the element does not exist until the country selection is clicked. I can't create the fixture in the phone field's constructor.

Reply:
I added a commit with some test code and comments I made while working through this issue. I removed them since I had found what I thought was a solid solution.

Reply:
@blackbaud-svossen Thanks for trying to work through this! Maybe I didn't see the `undefined` check you had in the `getCountryFixture()` before, so that certainly will cover my original concern. It also seems like your original solution avoids some nasty race conditions. Let's just resolve this and move on!